index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portofolio - Here we go</title>
+  </head>
+  <body>
+    <div id="app">
+      <header class="header">
+        <a href="#"><h2>My Logo</h2></a>
+        <a href="#"
+          ><img src="/img/icons/Union.png" alt="toggle menu icon"
+        /></a>
+      </header>
+
+      <div class="showcase">
+        <h3>I’m Joseph Glad to see you!</h3>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to
+          contactnxn me.
+        </p>
+        <a href="#"><p class="follow">let's connect</p></a>
+        <div class="social-media">
+          <a href="#"
+            ><img
+              src="/img/icons/twitter.svg"
+              alt="twitter"
+              title="follow me on twitter"
+          /></a>
+          <a href="#"
+            ><img
+              src="/img/icons/Linkedin_icon.svg"
+              alt="linkedin"
+              title="follow me on linkedin"
+          /></a>
+          <a href="#"
+            ><img src="/img/icons/medium.svg" alt="medium" title="follow me on"
+          /></a>
+          <a href="#"
+            ><img
+              src="/img/icons/github.svg"
+              alt="github"
+              title="follow me on github"
+          /></a>
+          <a href="#"
+            ><img
+              src="/img/icons/angellist.svg"
+              alt="angellist"
+              title="follow me on"
+          /></a>
+        </div>
```

Comment:
- [ ] As this is a list of social media links, kindly use the ul tag instead of a div, and also please wrap the anchors inside a li tag. And by the way, really great job on adding an alt tag to every image, that was great! 👏